src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/ThemeChange.cs

```diff
@@ -0,0 +1,31 @@
+#if ANDROID || IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class ThemeChange : _IssuesUITest
+	{
+		public override string Issue => "UI theme change during the runtime";
+
+		public ThemeChange(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public async Task AppThemeShouldChange()
+		{
+			_ = App.WaitForElement("label");
+
+			App.SetDarkTheme();
+			await Task.Delay(200);
```

Comment:
Instead of using a Task.Delay here can we watch for the theme change or use an appthemebinding (maybe on IsVisible) in the test app? And then do a WaitForElement with those? And then do the screenshot?